wsd/RequestVettingStation.hpp

```diff
@@ -48,14 +48,15 @@ class RequestVettingStation final : public std::enable_shared_from_this<RequestV
         : _requestDetails(requestDetails)
         , _poll(poll)
         , _mobileAppDocId(0)
+        , _logContextFD(-1)
     {
     }
 
     inline void logPrefix(std::ostream& os) const
     {
-        if (_socket)
+        if (_logContextFD > -1)
```

Comment:
Under-explained changed in the commit msg - but why not =)

Reply:
https://github.com/CollaboraOnline/online/pull/11756 to go with shared_ptr.lock() for this logging case